modules/textual_inversion/textual_inversion.py

```diff
@@ -39,6 +45,26 @@ def save(self, filename):
 
         torch.save(embedding_data, filename)
 
+    # Share the model to various locations on disk. Config directory defaults to
+    # /svr/stable-diffusion-webui-shared/embeddings-config.json
+    # [
+    #    {"name": "service 1", "path": "/knowlabs/stable-diffusion-webui-1/embeddings"},
+    #    {"name": "service 2", "path": "/knowlabs/stable-diffusion-webui-2/embeddings"},
+    #    {"name": "service 3", "path": "/knowlabs/stable-diffusion-webui-3/embeddings"},
+    #    {"name": "service 4", "path": "/knowlabs/stable-diffusion-webui-4/embeddings"}
+    # ]
+    def share(self):
+        if self.dirs:

```

Comment:
🤔  Not sure if this is a future PR or my brain is 🍳 , but where is the `dirs` being declared? May just for this simple PR change we can declare it when the class is initialized like so:

```python3
class Embedding:
    def __init__(self, vec, name, step=None, dir_path="embeddings_config.json"):
        // ... some code
        self.dirs = []
```

Reply:
hahahaha!!! Both of us are fried! 🍳

Reply:
Misspelled the field name.  Should be `self.dirs` not `self.dir`. Nice catch!

```python
def __init__(self, vec, name, step=None, dir_path="embeddings_config.json"):
         try:
             self.dir = json.load(dir_path)
         except Exception:
             self.dir = None
```